### PR TITLE
add tls options to redis

### PR DESCRIPTION
**Pull request by chrnie**
Add some options for tls to redis config template

### CODE REVIEW

**Comment by chrnie** [comment]
@cla-bot check

**Comment by chrnie** [comment]
@cla-bot check

**Comment by bobapple** [comment]
@cla-bot check

**Review by mkayontour — APPROVED** [review]

File: roles/icingadb_redis/defaults/main.yml [hunk 1]
```diff
@@ -60,3 +60,8 @@ icingadb_redis_hz: 10
 icingadb_redis_dynamic_hz: 'yes'
 icingadb_redis_aof_rewrite_incremental_fsync: 'yes'
 icingadb_redis_rdb_save_incremental_fsync: 'yes'
```
[the remark this answers is not in the document]
Comment (mkayontour) [reply roles/icingadb_redis/defaults/main.yml:62]: I was wrong - defaults where there before

**Pull request merged**